app/main.py

```diff
@@ -1,3 +1,55 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, second: Distance | int | float) -> any:
+        if isinstance(second, int) or isinstance(second, float):
```

Comment:
Change you approach by utilizing the `isinstance` syntaxis:
```suggestion
        if isinstance(second, (int, float)):
```
Apply this in all places where applicable.